frontend/src/shared/ui/select/select.tsx

```diff
@@ -43,23 +47,56 @@ export const Select = forwardRef<React.ElementRef<typeof Trigger>, SelectProps>(
       [options, returnValue, value],
     );
 
-    const label = useMemo<string>(() => {
+    const label = useMemo<string | null>(() => {
       if (isNull(selectedValue)) {
-        return placeholder;
+        return null;
       }
 
       if (displayValue) {
         return displayValue(selectedValue);
       }
 
       return selectedValue.label;
-    }, [displayValue, placeholder, selectedValue]);
+    }, [displayValue, selectedValue]);
+
+    const onChange = useCallback(
+      (newValue: string) => {
+        if (isStringEmpty(newValue)) {
```

Comment:
@worldspawn-web we need to add a lot a specific test cases, cuz this library was override our value by empty string on each choose.